src/views/poi_bloc/Images.jsx

```diff
@@ -0,0 +1,55 @@
+/* global _ */
+import React from 'react';
+import classnames from 'classnames';
+
+function createImage(param, classes) {
```

Comment:
This function which renders HTML is actually… a React component :) So it would be nicer to write it the right way, so you can use it like `<Image …>`.
And there is no problem to keep it in the same file if it's used only here.

Reply:
Oh good point! Not used enough to react I am...

Reply:
:) If you do that just pass `href` and `url` as separate props, so the low level component doesn't have to know how to access info.